src/sagemaker/inference_recommender/inference_recommender_mixin.py

```diff
@@ -149,12 +148,37 @@ def right_size(
 
         self._init_sagemaker_session_if_does_not_exist()
 
+        self.temp_model_name = None
```

Comment:
nit: the customer can see this `self.temp_model_name`, could we rename to smth like `self.right_sizing_model_name` to make it clear to the customer what this variable is for?